docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[uptime-set-up]]
+= Set up monitors
+
+++++
+<titleabbrev>Set up monitors</titleabbrev>
+++++
+
+To set up an uptime monitor, you will pull an Elastic-provided Docker image, configure the monitor,
+and use the Docker image to run the monitor and send data back to Elasticsearch.
+After setup is complete, the data will be available in the {uptime-app} in {kib} to view, analyze, and alert on.
+
+[discrete]
+[[uptime-set-up-choose]]
+== Choose an approach
+
+There are two ways to set up an uptime monitor: *{heartbeat}* or *{agent}*.
+
+[discrete]
+[[uptime-set-up-choose-heartbeat]]
+=== Heartbeat
+
+{heartbeat} is a lightweight daemon that you install on a remote server to periodically
+check the status of your services and determine whether they are available.
+Performance data is then gathered and reported back to {heartbeat},
+where it is formatted and sent to the {stack}.
+
+// What are the pros and cons?
+Using {heartbeat} works well when...
+
+The downsides of using {heartbeat} are...
+
+[discrete]
+[[uptime-set-up-choose-agent]]
+=== {agent}
+
+beta[] {agent} is a single, unified agent that you can deploy to hosts or containers to collect data and send it to the {stack}.
+Behind the scenes, {agent} runs {beats} or {elastic-endpoint} required for your configuration. 
+
+// What are the pros and cons?
+Using {agent} works well when...
+
+There are many benefits to using {agent} including central management, easier configuration processes, and endpoint protection (see more in {fleet-guide}/migrate-beats-to-agent.html#why-migrate-to-elastic-agent[Why migrate to Elastic Agent?]).
+However, the Elastic Synthetics integration is currently in beta so...
+
+The downsides of using {agent} are...
```

Comment:
What are the benefits and downside of using Elastic Agent?

Reply:
Elastic agent works better for users who want a more graphical interface for updating monitors. It is currently a beta product and in a certain amount of flux. It does not support custom location names, so is only suitable for running checks from a single location. It will be succeeded by [synthetics node](https://github.com/elastic/uptime/issues/441) in a few months.

For users who want to heavily script the creation and destruction of monitors it can be less convenient than a plain heartbeat.yml file, though we hope to offer more tooling around doing similar things with elastic agent / fleet in the future.